.github/workflows/rust_ci.yaml

```diff
@@ -51,6 +51,7 @@ jobs:
         run: |
           cargo install --force cargo-make
           cargo install --force duckscript_cli
+          cargo install cargo-ndk
```

Comment:
Is it necessary too?

Reply:
This was just a forced bypass, because cargo-ndk was not installed automatically with other packages.